Checksec.cpp

```diff
@@ -16,246 +19,261 @@ using json = nlohmann::json;
 
 namespace checksec {
 
-
-// RAII wrapper for LOADED_IMAGE
-class LoadedImage : public LOADED_IMAGE
-{
+class LoadedImage {
 public:
-    explicit LoadedImage(const std::string path)
-    {
-        if (!MapAndLoad(path.c_str(), NULL, get(), true, true)) {
-            throw ChecksecError("Couldn't load file; corrupt or not a PE?");
-        }
-    }
-    ~LoadedImage()
-    {
-        UnMapAndLoad(get());
+  explicit LoadedImage(const std::string path) {
+    if (!(pe_ = peparse::ParsePEFromFile(path.c_str()))) {
+      throw ChecksecError("Couldn't load file; corrupt or not a PE?");
     }
+  }
+  ~LoadedImage() { peparse::DestructParsedPE(get()); }
 
-    // can't make copies of LoadedImage
-    LoadedImage(const LoadedImage&) = delete;
-    LoadedImage& operator=(const LoadedImage&) = delete;
+  // can't make copies of LoadedImage
+  LoadedImage(const LoadedImage &) = delete;
+  LoadedImage &operator=(const LoadedImage &) = delete;
 
-private:
+  peparse::parsed_pe *get() { return pe_; }
```

Comment:
Hmm, let's get rid of `get()` entirely and add an `operator&` overload, so that we can do `&loadedImage` directly below.

Something like this should work:

```c++
  peparse::parsed_pe *operator&() { return pe_; }
```